markdown/blog/2023-mentorship-summary.md

```diff
@@ -0,0 +1,87 @@
+
+Now, let's see what the evaluation phase was like for selected participants. 
+
+#### What was the Evaluation Phase like?
+
+In the first evaluation phase, 11 candidates passed successfully, and two candidates were granted an extension to complete the phase along with the final evaluation. The second evaluation phase went smoothly, with a total of 12 candidates successfully passing. One reason for the successful second evaluation was our flexible evaluation period, allowing mentors to extend the deadline based on realistic timelines for their candidates.
+
+The second evaluation went well, with 12 candidates successfully passing the phase. One reason for the success of the second evaluation was that we did not enforce a really strict evaluation period. We defined a specific evaluation date but enabled mentors to extend it based on the date they feel is more realistic for their candidate.
+
+Throughout the evaluation phases, we received excellent feedback from mentors. They noted that certain mentees could develop their own solutions, think creatively, produce high-quality work, and demonstrate strong communication skills. During the first evaluation phase, some mentees had made significant progress, impressing us.
+
+It's important to mention that we evaluated both the mentees and the mentors during the phases. This approach allows us to provide feedback to mentors for improvement and better mentorship experiences in the future.
+
+
+#### Projects success
+
+During the program, eight projects ranging from engineering to design and writing got some pretty great help. Let's shine a light on some of these projects:
+
+- **[Website UI Kit design/dev project](https://github.com/asyncapi/design-system/issues/4)**: Thanks to the success of this program, the AsyncAPI website got a design system, which is something we really wanted. Now whenever someone works on the website, there's a consistent way of doing things design-wise. 
+- **[Simulator Desktop Application](https://github.com/asyncapi/community/issues/691)**:  The AsyncAPI simulator, which started as a project in GSoC '21 and was just a command-line tool, now has a version that works on desktops. Now it's easier for people to use it right from their computers. 
+- **[Documenting how different protocols work with AsyncAPI](https://github.com/orgs/asyncapi/discussions/533)**: A group of four candidates embarked on a detailed project documenting the interaction between various protocols and AsyncAPI. Thanks to the program, they developed and published two series of tutorials covering how to use AsyncAPI with the [Kafka]([https://www.asyncapi.com/docs/tutorials/websocket](https://www.asyncapi.com/docs/tutorials/websocket)) and [Websocket]([https://www.asyncapi.com/docs/tutorials/websocket](https://www.asyncapi.com/docs/tutorials/websocket)) protocols.
```

Comment:
btw the markdown links to docs that you have here look weird

in the end you should just have relative link like `/docs/tutorials/websocket`

just noticed that for kafka you point to websocket

Reply:
also maybe add that since the release of kafka section it got already `3559` and for websocket `2698` views

don't remember when they were released but probably few months ago